pkg/reconciler/identity/resources/iam_policy_bindings.go

```diff
@@ -68,17 +74,17 @@ func SetIamPolicy(ctx context.Context, action, projectID string, gServiceAccount
 	return nil
 }
 
-func MakeSetIamPolicyRequest(bindings []*iam.Binding, action, currentMember string) *iam.SetIamPolicyRequest {
+func makeSetIamPolicyRequest(bindings []*iam.Binding, action, currentMember string) *iam.SetIamPolicyRequest {
 	switch action {
-	case "add":
+	case Add:
 		return &iam.SetIamPolicyRequest{
 			Policy: &iam.Policy{
 				Bindings: append(bindings, &iam.Binding{
 					Members: []string{currentMember},
 					Role:    "roles/iam.workloadIdentityUser"}),
```

Comment:
nit: if you want add a constant to the role name, I see you are using it in two places.
but completely optional